README.md: Motivation example code fails compilation due to incorrect fixture name.

Compiling the first [Google.Test example](https://github.com/cpp-testing/GUnit/blame/master/README.md#L45) from README.md results in the following compilation error (with GCC 7.5.0 on Ubuntu 18.04):

```bash
/path/to/file.cpp:13:1: note: in expansion of macro ‘TEST_F’
 TEST_F(ExampleTest, ShouldReturnSumOf2Numbers) {
 ^~~~~~
/path/to/file.cpp:13:8: error: ‘ExampleTest’ was not declared in this scope
 TEST_F(ExampleTest, ShouldReturnSumOf2Numbers) {
        ^
```
The example declares a test fixture named `CalcFixture` but uses a different fixture name in TEST_F macro invocation.

--- a/README.md
+++ b/README.md
@@ -42,11 +42,11 @@
    std::unique_ptr<Calc> calc;                      |   }
   };                                                |
                                                     |   SHOULD("throw if division by 0") {
-  TEST_F(ExampleTest, ShouldReturnSumOf2Numbers) {  |     EXPECT_ANY_THROW(calc->div(42, 0));
+  TEST_F(CalcTest, ShouldReturnSumOf2Numbers) {  |     EXPECT_ANY_THROW(calc->div(42, 0));
     EXPECT_EQ(5, calc->add(4, 1));                  |   }
   }                                                 |
                                                     |   // TearDown
-  TEST_F(ExampleTest, ShouldThrowIfDivisionBy0) {   | }
+  TEST_F(CalcTest, ShouldThrowIfDivisionBy0) {   | }
     EXPECT_ANY_THROW(calc->div(42, 0));             |
   }                                                 |
   ```

